### PR TITLE
Added Option to Use OpenAi for Translations

### PR DESCRIPTION
**Pull request by BettinaMaria98**
- Updated webpack
- Added option for open AI Usage
- Added change on page (on save button and change value on form to trigger the "you really want to leave this page" alert on no save page realod

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,16 +18,35 @@ Module for auto translation using Google Cloud Translation for Silverstripe with
...
 
 ```yaml
 Level51\Autotranslate\AutotranslateFieldExtension:
+    translation_provider: 'google' # 'google' or 'openai' (default: google)
     google_cloud_translation_api_key: 'YOUR_API_KEY'
 ```
 
 See https://cloud.google.com/translate/docs/setup for setup instructions.
 
+## Setup OpenAI API
```
Comment (dkliemsch) [inline README.md:32]: A link to setup instructions for Open AI would be great

File: src/BlackListExtension.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    private static $db = [
+        'BlackListValues' => 'Text'
+    ];
+
+    public function updateCMSFields($fields)
+    {
+        $fields->addFieldsToTab('Root.Translation', [
+            TextareaField::create('BlackListValues', 'BlackList Values')
+                ->setDescription('Enter one value per line, the translatior will ignore these values and keep them as they are.'),
+            ReadonlyField::create('BlackListValuesPreview', 'Preview')
```
Comment (dkliemsch) [inline src/BlackListExtension.php]: What is the separate preview field for? Shouldn't the TextareaField be enough?

File: src/BlackListExtension.php [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+use SilverStripe\Forms\TextareaField;
+use SilverStripe\ORM\DataExtension;
+
+class BlackListExtension extends DataExtension
+{
+    private static $db = [
+        'BlackListValues' => 'Text'
+    ];
+
+    public function updateCMSFields($fields)
```
Comment (dkliemsch) [inline src/BlackListExtension.php:15]: English and German labels/descriptions would be nice :)

**Review by dkliemsch — CHANGES_REQUESTED** [review]
First of all: thanks for the great PR! 🙂 

I've left a few minor comments that could be resolved before merging.

File: src/BlackListExtension.php (hunk 2 above)
Comment (BettinaMaria98) [reply src/BlackListExtension.php]: This was just a box to check that everything works - but if you don't think it's necessary, I'll delete it. 🙂

**Comment by BettinaMaria98** [comment]
@dkliemsch Thanks for your review, I updated the files hope it is as expected 🙂

**Comment by dkliemsch** [comment]
@BettinaMaria98 Nice, thanks for the updates! I've just released a new version 0.3.0 🙂

**Pull request merged**